jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -140,9 +140,23 @@ public void assignGroupToRole(final String serverTemplateId, final String contai
     }
 
     @Override
-    public void assignGroupAndUserToRole(final String serverTemplateId, final String containerId, final String caseId, final String roleName, final String user, final String group) {
-        client.assignGroupToRole(containerId, caseId, roleName, group);
-        client.assignUserToRole(containerId, caseId, roleName, user);
+    public void assignToRole(final String serverTemplateId, final String containerId, final String caseId,
+                             final String roleName, final List<String> usersAssignments, final List<String> groupsAssignments,
+                             final List<String> oldUserAssignments, final List<String> oldGroupsAssignments) {
+
+        List<String> usersToRemove = new ArrayList<String>(oldUserAssignments);
```

Comment:
@cristianonicolai  I leave it into this method because when I moved it into presenter the case instance ((Void) -> findCaseInstance()) is called with any change in the assignments. Maybe we have to find another way for refreshing the assignments after all changes are done. What do you think?